react-delegationdashboard/src/components/Overview/Cards/index.tsx

```diff
@@ -0,0 +1,190 @@
+    return loginAddress === ownerAddress;
+  };
+
+  const getContractConfiguration = () => {
+    getContractConfig(dapp, delegationContract)
+      .then(value => {
+        if (isAdmin(value.returnData[0].asHex)) {
+          setIsAdminFlag(true);
+        }
+        let contractOverview = getContractOverviewType(value);
+        setContractOverview(contractOverview);
+      })
+      .catch(e => console.error('getContractConfig error ', e));
+  };
+
+  const getNumberOfActiveNodes = () => {
+    const query = new Query({
+      address: new Address(auctionContract),
+      func: new ContractFunction('getBlsKeysStatus'),
+      args: [Argument.fromPubkey(new Address(delegationContract))],
+    });
+    return new Promise<Array<NodeType>>(resolve => {
+      dapp.proxy
+        .queryContract(query)
+        .then(value => {
+          setNoNodes((value.returnData.length / 2).toString());
+        })
+        .catch(e => {
+          console.error('GetBlsKeysStatus error', e);
+        });
+    });
+  };
+
+  const getTotalStake = () => {
+    getTotalActiveStake(dapp, delegationContract)
+      .then(value => {
+        let input = value.returnData[0].asBigInt.toString();
+        let totalStake = denominate({
+          input,
+          denomination,
+          decimals,
+          showLastNonZeroDecimal: true,
+        });
+        setTotalActiveStake(totalStake || '0');
+      })
+      .catch(e => {
+        console.error('getTotalStake error ', e);
+      });
+  };
+  const getNetworkStake = () => {
+    dapp.apiProvider
+      .getNetworkStake()
+      .then(value => {
+        setNetworkStake(value);
+      })
+      .catch(e => {
+        console.error('getTotalStake error ', e);
+      });
+  };
+
+  React.useEffect(() => {
+    getNetworkStake();
+    getTotalStake();
+    getNumberOfActiveNodes();
+    getContractConfiguration();
+  }, []);
+
+  return (
+    <div className="cards d-flex flex-wrap mr-spacer">
+      <StatCard
+        title="Contract Stake"
+        value={totalActiveStake}
+        valueUnit={egldLabel}
+        color="orange"
+        svg="contract.svg"
+        percentage={`${getPercentage(
+          totalActiveStake,
+          denominate({
+            input: networkStake.TotalStaked.toString(),
+            denomination,
+            decimals,
+            showLastNonZeroDecimal: true,
+          })
+        )}% of total stake`}
+      />
+      <StatCard
+        title="Number of Nodes"
+        value={noNodes}
+        valueUnit=""
+        color="purple"
+        svg="nodes.svg"
+        percentage={`${getPercentage(
+          noNodes,
+          networkStake.TotalValidators.toString()
+        )}% of total nodes`}
+      />
+      <StatCard
+        title="Service Fee"
+        value={contractOverview.serviceFee || ''}
+        valueUnit="%"
+        color="red"
+        svg="service.svg"
+      >
+        {location.pathname === '/owner' && <SetPercentageFeeAction />}
```

Comment:
If we load this component from two different locations, like the owner/delegation path, this component can have an `isOwner` prop instead of checking for the location.pathname

Reply:
The idea here is to be shown only on the owner page even if the user is the owner